g2core/settings/settings_shapeoko2.h

```diff
@@ -54,6 +54,7 @@
 
 // Communications and reporting settings
 
+#define USB_SERIAL_PORTS_EXPOSED	2						// 1=single endpoint usb, 2=dual endpoint usb
```

Comment:
I havent had any issues, on the contrary its nice to be able to send ! around the queue, however I dont know if anyone else had problems. The only issue may be that sometimes the due needs to be physically (button push) reset before chili can connect.